indico/modules/rb/models/rooms.py

```diff
@@ -139,6 +139,15 @@ class Room(versioned_cache(_cache, 'id'), ProtectionManagersMixin, db.Model, Ser
     notification_before_days_monthly = db.Column(
         db.Integer
     )
+    notification_before_end_daily = db.Column(
+        db.Integer
```

Comment:
at least in newer code we explicitly specify `nullable=True` even though it's the default if omitted